Libraries/Utilities/Platform.ios.js

```diff
@@ -12,8 +12,21 @@
 
 'use strict';
 
+var invariant = require('invariant');
+
 var Platform = {
   OS: 'ios',
+  select: (obj: Object) => {
+    invariant(
+      obj && typeof obj === 'object',
+      'Platform.select: Must be called with an object'
+    );
+    invariant(
```

Comment:
Agreed with @ide, I would remove that invariant. Not specifying an android block if you are on ios is okay.